docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.5"
+
+networks:
+  metavoice-net:
+    driver: bridge
+
+volumes:
+  hf-cache:
+    driver: local
+
+services:
+  metavoice-server:
+    volumes:
+      - hf-cache:/.hf-cache
+      - ./assets:/app/assets
+    deploy:
+      replicas: 1
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
+              capabilities: [gpu]
+    runtime: nvidia
+    ipc: host
+    tty: true # enable colorized logs
+    build:
+      context: .
+    command: ['--huggingface_repo_id=metavoiceio/metavoice-1B-v0.1', "--port=80"]
+    ports:
+      - 8869:80
+    networks:
+      - metavoice-net
+    environment:
+      - NVIDIA_VISIBLE_DEVICES=all
+      - HF_HOME=/.hf-cache
+    logging:
+      options:
+        max-size: "100m"
+        max-file: "10"
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://metavoice-server:80/"]
+      interval: 1m
+      timeout: 20m
+      retries: 20
```

Comment:
Could you also add a ui service as shown here? https://github.com/metavoiceio/metavoice-src/compare/main...groovybits:metavoice-src:docker#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3